src/trix/core/helpers/custom_elements.coffee

```diff
@@ -18,9 +18,16 @@ insertStyleElementForTagName = (tagName) ->
   element = document.createElement("style")
   element.setAttribute("type", "text/css")
   element.setAttribute("data-tag-name", tagName.toLowerCase())
+  if nonce = getNonceForStyleElement()
+    element.setAttribute("nonce", nonce)
   document.head.insertBefore(element, document.head.firstChild)
   element
 
+getNonceForStyleElement = ->
+  if element = (document.head.querySelector("meta[name=trix-style-nonce]") ||
```

Comment:
Let's change this to `meta[name=trix-csp-nonce]` for better parity with `meta[name=csp-nonce]`